### PR TITLE
Adding manifest.json

### PR DESCRIPTION
**Pull request by TheChirpyWitch**
Issue#737

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	"src": "app/assets/images/icons/apple-touch-icon-120x120.png",
+	"sizes": "120x120",
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/apple-touch-icon-114x114.png",
+	"sizes": "114x114",
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/apple-touch-icon-precomposed.png",
+	/*"sizes": "144x144",*/
```
Comment (simonv3) [inline manifest.json]: This isn't valid JSON!

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	"src": "app/assets/images/icons/favicon-32x32.png",
+	"sizes": "32x32",
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/favicon-96x96.png",
+	"sizes": "96x96",
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/favico.ico",
+	/*"sizes": "16x16",*/
```
Comment (simonv3) [inline manifest.json]: Ditto

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/mstile-310x310.png",
+	"sizes": "310x310",
+	"type": "image/png"
+	}, {
+	"src": "app/assets/images/icons/mstile-70x70.png",
+	"sizes": "70x70",
+	"type": "image/png"
+	}]/*,
```
Comment (simonv3) [inline manifest.json]: Ditto

**Comment by simonv3** [comment]
Hey @TheChirpyWitch, thanks for the PR! I left a comment in there. Could you fix that and then I'll merge :)

**Comment by TheChirpyWitch** [comment]
Hey @simonv3 ! There is an option for giving the app url at the end. I haven't added it because I didn't know if I should make one or not.

File: manifest.json [hunk 4]
```diff
@@ -89,11 +89,11 @@
 	"src": "app/assets/images/icons/mstile-70x70.png",
 	"sizes": "70x70",
 	"type": "image/png"
-	}]/*,
+	}],
 	"related_applications": [{
 	"platform": "web"
 	}, {
 	"platform": "play",
```
Comment (simonv3) [inline manifest.json]: Yeah we don't have a play app, so you can just remove this entire object.

**Comment by simonv3** [comment]
Awesome! Thanks @TheChirpyWitch. In a couple of minutes we should be able to test whether it has any effect on an Android phone.

**Comment by TheChirpyWitch** [comment]
Fingers crossed. :grin:

**Pull request merged**